scene/resources/bit_map.cpp

```diff
@@ -704,6 +704,87 @@ void BitMap::blit(const Vector2i &p_pos, const Ref<BitMap> &p_bitmap) {
 	}
 }
 
+void BitMap::bitwise_and(const Ref<BitMap> &p_other) {
+	ERR_FAIL_COND_MSG(p_other.is_null(), "Null reference to supplied BitMap other.");
```

Comment:
Reword it to sound more natural:

```suggestion
	ERR_FAIL_COND_MSG(p_other.is_null(), "The BitMap passed as the \"other\" parameter is null.");
```

(Same for the other identical messages further below.)